etrago/appl.py

```diff
@@ -449,10 +449,15 @@ def etrago(args):
 
     # scenario decommissioning
     if args['scn_decommissioning'] is not None:
-        network = decommissioning(
-            network,
-            session,
-            args)
+        for i in range(len(args['scn_decommissioning'])):
+            network = decommissioning(
+                    network,
+                    session,
+                    version=args['gridversion'],
+                    scn_decommissioning=args['scn_decommissioning'][i],
+                    start_snapshot=args['start_snapshot'],
```

Comment:
Hi, here you define start_snapshot and end_snapshot for decommissioning(), but these arguments are not required by the function which leads to an error.

Reply:
Hi, thank you for your help. I fixed this in the last commit.